niche_vlaanderen/nutrient_level.py

```diff
@@ -40,7 +40,7 @@ def __init__(
                 ["system_tables"], "nutrient_level.csv")
 
         self.ct_lnk_soil_nutrient_level = pd.read_csv(ct_lnk_soil_nutrient_level)
-        self._ct_management = pd.read_csv(ct_management).set_index("management")
+        self._ct_management = pd.read_csv(ct_management) #.set_index("management")
```

Comment:
If the code is no longer required, remove it instead of changing it to inline documentation